### DISCUSSION TITLE
Insufficient keyboard support for embedded player widget

### DISCUSSION THREAD

**Discussion by stevefrenzel** [tidal-music/discussions#55] [Ideas & Feature requests]
Hi there,

I recently did a screen reader check for the embedded player widget and while it's pretty usable (at least for me), there are some serious issues regarding keyboard support.

You can find more details in my blog post: [Screen reader check: TIDAL widget](https://stevefrenzel.dev/posts/screen-reader-check-tidal-widget/)

Short summary: Play button doesn't exist in the accessibility tree and therefore cannot be operated via keyboard. It did work when using screen readers NVDA and Narrator on Windows 11, though.

**Comment by stefanmoro** [comment]
Hey @stevefrenzel, and thanks for the feedback.

We're currently doing some updates to the embed player, and I'll forward this to the team handling the embed player. CC: @enjikaka @osmestad
